data/remnant ships.txt

```diff
@@ -237,7 +239,7 @@ ship "Gull"
 		"fuel capacity" 600
 		"ramscoop" 1
 		"cargo space" 175
-		"outfit space" 310
+		"outfit space" 316
```

Comment:
Ship | Extra outfit space 
------|-------
Gull | 55
Gull (Bunks) | 33
Gull (Cargo) | 33
Gull (Heavy) | 5
Gull (Sniper) | 3 
Gull (Support) | 1

Three of the six variants already have room for salvage scanners. The other three could very easily install an outfits expansion, as they don't have any cooling installed to begin with that would be harmed, and the loss in heat capacity due to the reduction in heat dissipation is in the low double digits compared to the four-digit heat capacity. If that's a concern, simply install a cooling outfit as well.